src/components/FullResourcePage.tsx

```diff
@@ -0,0 +1,170 @@
+import axios from "axios";
+import { useState } from "react";
+import { useParams } from "react-router";
+import { BackendURL } from "../utils/backendURL";
+import useFetchResources from "../utils/useFetchResources";
+import "./FullResourcePage.css";
+import useFetchComments from "../utils/useFetchComments";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+//--------------------------------------------------------------------------------------JSX Element declaration
+export default function FullResourcePage(): JSX.Element {
+  const { resources } = useFetchResources();
+
+  const [commentSubmit, setCommentSubmit] = useState({
+    user_id: 0,
+    comment_text: "",
+  });
+
+  //--------------------------------------------------------------------------------------Defining selected resource
+  const { id } = useParams();
+  const resource = resources.find((resource) => {
```

Comment:
what is const resource? avoid giving two variables the same name its used in parenthesis in the find function and the variable declaration